smoketest/tests/send_tokens.rs

```diff
@@ -71,19 +64,10 @@ async fn send_tokens() {
         .unwrap();
     assert_eq!(receipt.status.unwrap().as_u64(), 1u64);
 
-    let recipient = MultiLocation {
-        parents: 0,
-        interior: X1(Junction::AccountId32 {
-            network: None,
-            id: FERDIE,
-        }),
-    }
-    .encode();
-
     // Lock tokens into vault
     let value1: u128 = U256::from(value).low_u128();
```

Comment:
Remember I met `Insufficient***` error for launch this test multiple times, so would suggest  use a lower value here. Meanwhile some change in https://github.com/Snowfork/snowbridge/pull/905/files#r1276100691

Reply:
Resolved in [0c4bdcc](https://github.com/Snowfork/snowbridge/pull/904/commits/0c4bdccbe776bfeb53b2d24c4437b4a0c74b6fa2)